### PR TITLE
Escape # in filename url

### PR DESCRIPTION
**Pull request by isuruf**
git+https://github.com/org/project#egg=project can have # in the name and it should be escaped when used in a URL

### CODE REVIEW

File: asv/www/asv.js [hunk 1]
```diff
@@ -239,6 +239,7 @@ $(document).ready(function() {
                          "LPT2", "LPT3", "LPT4", "LPT5", "LPT6", "LPT7", "LPT8",
                          "LPT9"];
         name = name.replace(bad_re, "_");
+        name = name.replace("#", "%23");
```
Comment (pv) [inline asv/www/asv.js]: Also take the comment above "The implementation must match asv.util.sanitize_filename" into account.

Comment (isuruf) [reply asv/www/asv.js]: This is just escaping the # character. Filename is the same.

Comment (pv) [reply asv/www/asv.js]: Or, possibly, this is not the right place for the url quoting, but it should be done at the point where the json request is made.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**